### PR TITLE
add links.js file with all links

### PR DESCRIPTION
**Pull request by fMercury**
Move all hyperlinks (on [Header](https://github.com/fMercury/dxdao-landing-page/pull/6/files#diff-6254cc23a9bd1078e60324425e7b2cb9), [LandingPage](https://github.com/fMercury/dxdao-landing-page/pull/6/files#diff-40fc2364e3e04b62909bc22ef216c6b5) and [Footer](https://github.com/fMercury/dxdao-landing-page/pull/6/files#diff-f2c0336d5382cf04e171399da5055bc3) ) to [links](https://github.com/fMercury/dxdao-landing-page/pull/6/files#diff-39c95eeb1ec1fd9bcb5b5dcfa040257f) file

In addition, this PR solves
#315
#319

### CODE REVIEW

File: src/links.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Landing Page
```
Comment (AugustoL) [inline src/links.js:1]: please export only a single json object, not link by link.

File: src/components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -344,204 +367,261 @@ const LandingPage = () => {
...
+                                <ProductNameWrapper>
+                                    <ProductLogo src="Dxswap.svg" />
+                                    <ProductName>DXswap.eth</ProductName>
+                                </ProductNameWrapper>
+                                <ProductDescription>
+                                    DXswap is a protocol for exchanging ERC-20 
+                                    tokens.
+                                </ProductDescription>
+                            </ProductPanel>
+                        </ProductClickable>
```
Comment (AugustoL) [inline src/components/LandingPage/LandingPage.js:516]: Why so much space between this lines? Dont use breaklines between html elements.

File: src/components/LandingPage/LandingPage.js [hunk 3]
```diff
@@ -344,204 +367,261 @@ const LandingPage = () => {
...
-            </AboutUsSection>
-            <JoinSection>
+
+                <BannerSection>
+
+                        <TagLine>We decentralize DeFi</TagLine>
+
+                    <BannerMessage>
+
+                        <HeavyMessage>DXdao </HeavyMessage>is a{' '}
```
Comment (AugustoL) [inline src/components/LandingPage/LandingPage.js:379]: Why the `is a{' '}` ? We want to show the description here.

Comment (fMercury) [reply src/components/LandingPage/LandingPage.js:379]: that item was not changed from the original version : [levelkdev/BC-DAPP/src/components/LandingPage/LandingPage.js](https://github.com/levelkdev/BC-DAPP/blob/master/src/components/LandingPage/LandingPage.js)

**Pull request merged**